CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+2023-09-25 v2.7:
+
+* add: compatibility with PostgreSQL 15 and 16, Jehan-Guillaume de Rorthais, Thomas Reiss
```

Comment:
I think it's missing entries for cb0e5f477986bf7b4ba4eb0a741b6a03c7b23e80 and 59fd2b9084aa4ece33316c833c3aa4124ad77ec2.

Reply:
indeed for cb0e5f4, but 59fd2b9 is listed as `change: simplify session accounting in `backends_status`, Thomas Reiss`

Reply:
fixed in next commit